### PR TITLE
#163050304 create rsvp for a meetup

### PR DESCRIPTION
**Pull request by Mupa1**
__What does this PR do?__
It enables a user to rsvp for a meetup

__Descriptions of the tasks to be completed?__
Have the following endpoint working
```POST /api/v1/meetup/1/rsvp```

__How should this be manually tested?__
* After cloning the repo cd into it and RUN flask
* Run tests using pytest
* Using Postman test the endpoint POST /api/v1/meetup/1/rsvp to rsvp for a meetup(yes, no, maybe)

__Relevant pivotal tracker stories__
https://www.pivotaltracker.com/n/projects/2235450
#163050304

__Screenshot__
screenshot from 2019-01-16 22-09-46

### CODE REVIEW

File: app/api/v1/rsvp/model.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        """method to get all rsvp"""
+        return self.db
+
+    def find(self, rsvp_id):
+        """method to get a specific rsvp"""
+        for rsvp in self.db:
+            if rsvp['id'] == rsvp_id:
+                return rsvp
+        
+        return "rsvp does not exist"
```
Comment (Ogollah) [inline app/api/v1/rsvp/model.py]: Add a new line here

File: app/api/v1/rsvp/views.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        if rsvp == "rsvp does not exist":
+            return jsonify({
+                "status" : 404,
+                "error": "rsvp does not exist"
+            })
+        
+        return jsonify({
+            "status" :200,
+            "data" : rsvp
+        })
```
Comment (Ogollah) [inline app/api/v1/rsvp/views.py]: Add a new line year

**Comment by Ogollah** [comment]
Good job, but, try to fix coveralls if its possible.

**Comment by jkamz** [comment]
This looks good. Keep up

File: app/api/v1/rsvp/model.py (hunk 1 above)
Comment (Mupa1) [reply app/api/v1/rsvp/model.py]: Done

File: app/api/v1/rsvp/views.py (hunk 2 above)
Comment (Mupa1) [reply app/api/v1/rsvp/views.py]: Done

**Comment by Mupa1** [comment]
@Ogollah i will try and fix coveralls
@jkamz thanks

**Pull request closed without merging**